src/vm/hints/blake2s_utils.rs

```diff
@@ -71,19 +71,19 @@ fn compute_blake2s_func(
    compute_blake2s_func(segments=segments, output_ptr=ids.output)
 */
 pub fn compute_blake2s(
-    vm: &mut VirtualMachine,
+    variables: HintVisibleVariables,
     ids: &HashMap<String, BigInt>,
     hint_ap_tracking: Option<&ApTracking>,
```

Comment:
We can probably include `ids` and `hint_ap_tracking` in the new structure.

Reply:
Will get to this on a later phase of the refactor